packages/mds-db/processors.ts

```diff
@@ -117,9 +117,9 @@ export async function insertMetrics(metric: MetricsTableRow) {
 interface GetAllMetricsArgs {
   start_time: Timestamp
   end_time: Timestamp
-  provider_id: UUID | null
+  provider_id: UUID[]
```

Comment:
If these are going to be arrays, pluralize to `provider_ids`

Reply:
I think we want to hammer out how this is passed first.

Reply:
Like, you think there's still a chance the ultimate type for this could be `UUID | UUID[]`?

Reply:
Yes, unfortunately. Apparently it's some kind of REST semantic.

Reply:
Jk, these should be pluralized

Reply:
I saw that, but I think it would be cleaner if calls to `getAllMetrics` could assume they were getting an array of UUIDs.